docs/tasks/administer_cluster_resources.md

```diff
@@ -0,0 +1,100 @@
+# Administer Cluster resources
+
+This page shows you how to represent your cluster resource limits and to
+establish fair sharing rules among the tenants.
+
+The intended audience for this page are cluster administrators.
+
+## Before you begin
+
+You need to have a Kubernetes cluster, the kubectl command-line tool
+must be configured to communicate with your cluster, and [Kueue installed](/README.md#installation).
+
+## Single Cluster Queue setup
+
+### 1. Create a [Cluster Queue](/docs/concepts/cluster_queue.md)
+
+Create a single ClusterQueue to represent the resource limits for your
+entire cluster.
```

Comment:
I think we need to talk "quota" rather than limits of an entire cluster so we don't give the illusion that Kueue will have full control over the cluster and pods created outside queued workloads.